packages/perennial/deploy/005_deploy_multiinvoker.ts

```diff
@@ -47,13 +56,22 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     contract: 'TransparentUpgradeableProxy',
     args: [multiInvokerImpl.address, proxyAdminAddress, '0x'],
     from: deployer,
-    skipIfAlreadyDeployed: true,
+    skipIfAlreadyDeployed: false,
     log: true,
     autoMine: true,
   })
 
   // INITIALIZE
-  const multiInvoker = await new MultiInvoker__factory(deployerSigner).attach((await get('MultiInvoker_Proxy')).address)
+  let multiInvoker: any
+  if (!rollup) {
```

Comment:
Any reason we need this conditional? The important part of the interface used below is the same for both of these so I think we can just keep the original